playbooks/gce/openshift-node/config.yml

```diff
@@ -12,6 +12,11 @@
   hosts: "tag_env-host-type-{{ oo_env }}-openshift-master"
   connection: ssh
   user: root
+  gather_facts: yes
+  tasks:
+    - command: 'curl "http://metadata.google.internal/computeMetadata/v1/instance/network-interfaces/0/access-configs/0/external-ip" -H "Metadata-Flavor: Google"'
+      register: output
+    - set_fact: gce_public_ip="{{ output.stdout }}"
```

Comment:
You're right, it's always there after dynamic inventory runs.

Here's the sequence:
1. Nothing exists
2. dynamic inventory runs, gets nothing back
3. we launch the master instance
4. We use 'add_host' to manually add the new master instance to the group:  tag_env-host-type-{{ oo_env }}-openshift-master
5. we launch the node instances
6. we're configuring the node instances with the master instance information.
7. since the inventory hasn't run after the master instance was created, then gce_public_ip wasn't populated for the instance, so we have to manually populate it here.

This is only a problem during initial cluster create. During a normal config, you are correct. Since both cases use the same config.yml playbook (by design), we have to account for both use cases, which this line does.

In the future, we may write a gce_facts module (akin to the ec2_facts module) to make this cleaner. But for now, this works reliably.